src/Symfony/Component/VarDumper/Caster/ExceptionCaster.php

```diff
@@ -83,6 +84,15 @@ public static function castThrowingCasterException(ThrowingCasterException $e, a
         return $a;
     }
 
+    public static function castThrownExceptionDetails(ThrownExceptionDetails $exceptionDetails, array $a, Stub $stub, bool $isNested, int $filter = 0)
```

Comment:
Do we need this method?

Without it it would fallback to `castException`, right? Would that render much differently?

Reply:
It will not fallback to `castException`, because `ThrownExceptionDetails` does not extend from `Exception`. Without it, it would not display a nice trace, and the `ThrownExceptionDetails` class will be shown as the name of the exception class, instead of the actual exception class.

Reply:
Yeah. I just found that out be experimenting. 

Thank you for the explanation.